frontend/app/(tabs)/home.tsx

```diff
@@ -29,24 +29,14 @@ const Index = () => {
   const SCROLL_PADDING = 20;
 
   const searchAnimation = useRef(new Animated.Value(0)).current;
+  const searchTimeout = useRef<NodeJS.Timeout>();
```

Comment:
Do we need this? I think we can deal with this as we do with the normal API call, let's remove this.